source/js/components/expander.js

```diff
@@ -20,17 +20,23 @@ var Panel = React.createClass({
     this.props.activateKey(itemKey);
   },
   render: function() {
+    var expanded = this.props.itemKey === this.props.activeKey;
+    var style = {
+      height: "0"
+    };
     var itemClassName = classnames("expander-item", {
-      "expander-item-active": this.props.itemKey === this.props.activeKey
-    });
-    var contentClassName = classnames("expander-content", {
-      "expander-content-active": this.props.itemKey === this.props.activeKey
+      "expander-item-active": expanded
     });
+    if (this.content && expanded) {
```

Comment:
this.content is set after the first render, so it's initially undefined. It needs to wait until it's rendered before it can have a reference.